### ISSUE TITLE
Faltan resoluciones

### ISSUE THREAD

**Issue by doorgan** [Nekitaw/ubeat-web#1]
Faltan resoluciones como 1366x768, que es una resolución estandar en monitores de 19"

**Comment by Nekitaw** [comment]
seguro?

https://k60.kn3.net/E/4/A/8/0/F/49A.png

**Comment by doorgan** [comment]
Sí, me acabo de fijar y muestra 1360x768[Windowed] pero no 1366x768

**Comment by Nekitaw** [comment]
Tengo que ver, quizás sea XNA o no se me ocurre otra, de todas formas, ahora estoy porteando a Monogame
